project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject com.novemberain/pantomime "2.9.0-SNAPSHOT"
```

Comment:
Please don't mess with project version numbers. In fact, never do that. It's not up to you to decide what the next version should be: maintainers will bump as necessary.